lib/main.dart

```diff
@@ -2,18 +2,62 @@ import 'package:flutter/material.dart';
 
 void main() {
   var app = MaterialApp(
-      home: Scaffold(
-    appBar: AppBar(
-      title: Text('INAMOVIBLE'),
-    ),
-    body: Container(
-      child: Center(
-          child: Text(
-        "HELLO PUTOS",
-        style: TextStyle(fontStyle: FontStyle.italic),
+    home: Scaffold(
+      appBar: AppBar(
+        title: Text('Entrenadores'),
+      ),
+      body: Container(
+          child: Column(
+        children: <Widget>[
+          TrainerCard(
+              nombre: "Octavius",
+              ambito: "Entrenador Olimpico del propio Olimpo"),
+          TrainerCard(
+              nombre: "Erick von Kaiserdom",
+              ambito: "Entrenador del Kaiserdom"),
+          TrainerCard(nombre: "Rafa", ambito: "Entrenador de los kapes"),
+          TrainerCard(nombre: "Rafa", ambito: "Entrenador de killos"),
```

Comment:
```suggestion
          TrainerCard(nombre: "Rafa", ambito: "Entrenador de vagos"),
```
Decir killos es despectivo